### PR TITLE
attach: improve JAWS readability

### LINKED ISSUE #1491: attach: improve JAWS readability
JAWS should read
- [x] `attach` as "button"
- [x] `attach_disabled` as "button unavailable"
- [x] `button` field
- [x] `noFileText` field

Check and fix divergencies.

### PR DESCRIPTION
**Pull request by sipayRT**
Closes #1491

### CODE REVIEW

**Comment by sipayRT** [comment]
@aristov this branch was rebased. need your review

File: common.blocks/attach/__no-file/attach__no-file.bh.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 module.exports = function(bh) {
     bh.match('attach__no-file', function(ctx) {
-        ctx.tag('span');
+        ctx
+            .tag('span')
+            .attr('aria-hidden', true);
```
Comment (qfox) [inline common.blocks/attach/__no-file/attach__no-file.bh.js]: Incompatible with BH 4.1+, Use `'true'` instead of `true` ;-) upd: Or fix tests.

Comment (sipayRT) [inline common.blocks/attach/__no-file/attach__no-file.bh.js]: I'll do it in common a11y branch

**Comment by aristov** [comment]
JAWS speaks native "browse" and "no file selected" regardless of `button` and `noFileText` fields. Try to hide native control from JAWS by using `role="presentation"` or `aria-hidden="true"`.

**Comment by aristov** [comment]
`noFileText` field still not accessible.

**Comment by sipayRT** [comment]
@aristov 

> Try to hide native control from JAWS

I tried to do it, but in case when some file was chosen, you can't prevent reading of this element (even `aria-hidden` didn't help). So I offer to disable elem `__text` to avoid double-reading of chosen filename.

**Pull request merged**